Sources/Benchmark/Benchmark.swift

```diff
@@ -41,12 +41,23 @@ public final class Benchmark: Codable, Hashable {
 
     /// Alias for closures used to hook into setup / teardown
     public typealias BenchmarkHook = () async throws -> Void
+    public typealias BenchmarkSetupTeardownHook = BenchmarkHook
+
+    #if swift(>=5.8)
+        @_documentation(visibility: internal)
+    #endif
+    public static var startupHook: BenchmarkSetupTeardownHook? // Should be removed when going to 2.0, just kept for API compatiblity
```

Comment:
Add a warning about deprecated API?

Reply:
Yeah, I did first do deprecation annotation, but then everyone gets it (even people don't using the hook) which is bad, as we still need to call it, so I just hid it from the documentation so there'll be no new takes on the obsolete naming and then we'll do a break someday...